.github/workflows/tests.yml

```diff
@@ -77,6 +77,11 @@ jobs:
             condaforge: 1
             pytest-extra-options: "-W ignore::ImportWarning -W ignore::DeprecationWarning:cvxpy.interface.scipy_wrapper"
 
+          # Windows
+          - case-name: Windows Latest
+            os: windows-latest
+            pytest-extra-options: "-k 'not (test_correlation or test_interpolate or test_dnorm)'"
```

Comment:
I would prefer to skip `dnorm` tests from the test files instead of here.
For `test_correlation` and `test_interpolate`, if it's not that they don't pass, but that they are very slow, skipping them here seems appropriate.